zerver/migrations/0467_rename_followup_day_email_templates.py

```diff
@@ -0,0 +1,95 @@
+# Generated by Django 4.2.2 on 2023-07-11 10:45
+
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.db.models import F, Func, JSONField, TextField, Value
+from django.db.models.functions import Cast
+
+# ScheduledMessage.type for onboarding emails from zerver/models.py
+WELCOME = 1
+
+
+def update_for_followup_day_email_templates_rename(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    ScheduledEmail = apps.get_model("zerver", "ScheduledEmail")
+
+    account_registered_emails = ScheduledEmail.objects.annotate(
+        as_jsonb=Cast("data", JSONField())
+    ).filter(type=WELCOME, as_jsonb__template_prefix="zerver/emails/followup_day1")
+    account_registered_emails.update(
+        data=Cast(
+            Func(
+                F("as_jsonb"),
+                Value(["template_prefix"]),
+                Value("zerver/emails/account_registered", JSONField()),
+                function="jsonb_set",
+            ),
+            TextField(),
+        )
+    )
```

Comment:
Yeah, agreed that'd be a nice tweak, though as a migration we'll run once, it's not absolutely critical.

I'm not gonna block merging on this, since it's good to get migrations in to avoid additional future renumbering pain, but it is certainly reasonable to do a quick follow-up PR to tweak thios.